website/docs/Examples/Integrate - OpenAI.md

```diff
@@ -0,0 +1,183 @@
+FLAML has integrated the OpenAI's completion API.
```

Comment:
Readers without background of FLAML could be reading this if they visit this page from reading the paper. Please think of what to say here to keep them interested.